templates/Includes/GitDetailedBuildReference.ss

```diff
@@ -3,46 +3,64 @@
 		<% if $BareURL %>$BareURL<% else %>$URL<% end_if %>
 	</a>
 </h3>
+<% if $Environment.CurrentBuild %>
 
-<div class="row">
-	<div class="col-md-6">
-		<ul class="build-details">
-			<% if $Environment.Project.RepositoryInterface.CommitURL %>
-				<li>
-					<% if $Tags %>
-						<% loop $Tags %>
-							<span class="deployment-tag">$Me</span>
-						<% end_loop %>
+	<div class="row">
+		<div class="col-md-6">
+			<a href="#" id="current-build-toggle">Current Build Details <i class="fa fa-caret-down"></i></a>
+			<div class="table-responsive hide current-build-data">
+				<table class="table">
+					<% if $Environment.CurrentBuild.DeploymentStrategy %>
```

Comment:
Should probably check `$Environment.CurrentBuild.DeploymentStrategy.getChange("Infrastructure version")` otherwise this will show up for environments that don't have that change in the strategy, e.g. Capistrano envs.